apps/mosaico/src/Mosaico.purs

```diff
@@ -606,5 +627,6 @@ render setState state components router onPaywallEvent =
       router.pushState (write {}) $ "/sök?q=" <> query
 
     simpleRoute path = do
+      runEffectFn1 refreshAdsImpl ["mosaico-ad__top-parade", "mosaico-ad__parade"]
```

Comment:
I'm not quite sure about having this call in this place. Would `routeListener` be better place? That's guaranteed to be called whenever user navigates in any way.

But if this is the way it should work then this place is fine.

Reply:
I think that's a good idea. I'll look at it.